components/lane_follower/src/individual_follower.py

```diff
@@ -0,0 +1,150 @@
+import math
+
+import cv2
+import numpy as np
+import toml
+from einops import repeat
+
+
+class ifResult:
+    def __init__(self):
+        self.result_img = None
+        self.empty_windows = 0
+        self.heading = 0
+        self.slope = 0
+
+    pass
+
+
+class IndividualFollower:
+
+    def __init__(self, toml_path='Follower_Consts.toml'):
+
+        self.consts = toml.load(toml_path)["Individual_Follower"]
+        # self.fit is set in Plot_Line
+        self.fit = None
+        # These two are set below
+        self._binary_warped = None
+        self._histogram = None
+
+    # Need to determine if these should remain properties
+    # Or if they should be passed as arguments to Plot_Line
+    @property
+    def _binary_warped(self, value: np.ndarray):
+        self._binary_warped = value
+        self._histogram = np.sum(
+            self.binary_warped[self.binary_warped.shape[0] // 2 :, :], axis=0
+        )
+
+    def get_white_pixels(self):
+        # Returns an output image
+        NWINDOWS = self.consts["NWINDOWS"]
+        SEARCH_WIDTH = self.consts['SEARCH_WIDTH']
+
+        window_height = np.int32(self._binary_warped.shape[0] / NWINDOWS)
+        lane_inds = []
+
+        nonzero_pixels = self._binary_warped.nonzero()
+        nonzero_y = np.array(nonzero_pixels[0])
+        nonzero_x = np.array(nonzero_pixels[1])
+
+        empty_windows = 0
+        lane_base = np.argmax(self.histogram[:])
+
+        # Image to visualize output
+        out_img = repeat(self._binary_warped, 'h w -> h w c', repeat=3) * 255
+        ##These outputs need to be confirmed compatible
+        # out_img = (
+        #     np.dstack(
+        #         self._binary_warped, self._binary_warped, self._binary_warped
+        #     )
+        #     * 255
+        # )
+
+        for window in range(NWINDOWS):
+            window_dims = window * window_height
```

Comment:
If you went through the effort of making `window_height` an `np.int32`, I think you should do the same with `window`, or make them both Python `int`s.